wasm.md

```diff
@@ -661,13 +661,16 @@ Currently, only one memory may be accessible to a module, and thus the `<memAddr
 
 ```k
     syntax Instr ::= "(" "memory"               ")"
-                   | "(" "memory" Int           ")" // Max only
+                   | "(" "memory" Int           ")" // Size only
                    | "(" "memory" Int Int       ")" // Min and max.
                    |     "memory" "{" Int MemBound "}"
  // --------------------------------------------------
-    rule <k> ( memory                 ) => memory { 0   .MemBound } ... </k>
-    rule <k> ( memory         MAX:Int ) => memory { 0   MAX       } ... </k>
-    rule <k> ( memory MIN:Int MAX:Int ) => memory { MIN MAX       } ... </k>
+    rule <k> ( memory                 ) => memory { 0                 .MemBound         }         ... </k>
+    rule <k> ( memory MIN:Int         ) => memory { MIN               .MemBound         }         ... </k>
+      requires MIN <=Int #maxMemorySize()
+    rule <k> ( memory MIN:Int MAX:Int ) => memory { MIN               MAX               }         ... </k>
```

Comment:
Why is there so much extra horizontal whitespace on the RHS of these rules? Can we squish it down a bit?

Reply:
Oh, right, it's due to the rule coming underneath it (part of next PR), which is pretty wide. I'll fix it, just need to rebase a little.